gips/data/landsat/landsat.py

```diff
@@ -200,6 +220,33 @@ def __init__(self, filename):
                 }
             self.visbands = [col for col in smeta['colors'] if col[0:4] != "LWIR"]
             self.lwbands = [col for col in smeta['colors'] if col[0:4] == "LWIR"]
+        elif c1_match:
+            VerboseOut('C1 asset', 2)
+
+            self.tile = c1_match.group('path') + c1_match.group('row')
+            year = c1_match.group('acq_year')
+            month = c1_match.group('acq_month')
+            day = c1_match.group('acq_day')
+            self.date = datetime.strptime(year + month + day, "%Y%m%d")
+
+            self.asset = 'C1'
+            self.sensor = "L{}{}".format(c1_match.group('sensor'), int(c1_match.group('satellite')))
```

Comment:
Is there an issue with the number of characters in `self.sensor` varying?  If the satellite group's two characters can have a leading zero, when you `int()` it could go down to one.  Maybe that's intentional; if so nevermind.  : ]